doc/changes/changes_1.5.0.md

```diff
@@ -0,0 +1,42 @@
+# Exasol Release Droid 1.5.0, released 2023-??-??
+
+Code name: Updated Release Guide
+
+## Summary
+
+This release fixes and improves the Release Guide added in version 1.4.0.
+
+## Features
+
+* #281: For projects with multiple error codes display the shortest common prefix.
+
+## Bugfixes
+
+Marked vulnerability CVE-2022-45688 in [org.json:json:jar:20220924](https://ossindex.sonatype.org/component/pkg:maven/org.json/json@20220924?u) in compile to be ignored as affected releases to community portal are currently disabled anyway.
```

Comment:
Possible options: replace it with Jakarta JSON or remove the code ;)

Reply:
I will try to replace in the scope of ticket #280